Block/System/Config/Button/Import.php

```diff
@@ -114,14 +116,24 @@ public function getImport()
         return $this->import;
     }
 
+    /**
+ * Order Import model
+ *
+ * @return \Metrilo\Analytics\Helper\OrderSerializer
+ */
+    public function getOrderData()
```

Comment:
Why do you need these methods? Can't you get the `orderData` by using `->` operator?